catalog/harvest/catalog-harvest-metacardadaptor/src/main/java/org/codice/ddf/catalog/harvest/adaptor/MetacardOnlyAdaptor.java

```diff
@@ -129,19 +129,20 @@ public void update(HarvestedResource resource, String updateId) throws HarvestEx
             throw new HarvestException(
                 String.format("Failed to update resource [%s]", resource.getName()), e);
           }
+
+          return null;
         });
   }
 
   @Override
   public void delete(String id) {
-    DeleteRequest deleteRequest = new DeleteRequestImpl(id);
+    final DeleteRequest deleteRequest = new DeleteRequestImpl(id, getSecurityMap());
```

Comment:
This will probably need to be tested, but I believe there are issues passing an unmodifiable map into the Catalog framework.